src/routes/events.ts

```diff
@@ -0,0 +1,35 @@
+import express, { NextFunction, Request, Response } from "express";
+import { validationResult } from "express-validator/check";
+import { cache } from "../middleware/cache";
+import { fetchEvents } from "../utils/fetchUtils";
+import {
+  validateLanguageParam,
+  validateUserGroupIdParam,
+} from "../utils/validateParamsUtils";
+
+const router = express.Router();
+router.use(cache);
+
+router.get(
+  "",
+  [
+    validateLanguageParam(),
+    validateUserGroupIdParam(),
+  ],
+  (req: Request, res: Response, next: NextFunction) => {
+    validationResult(req).throw();
+
+    const userGroupId: number = req.query.userGroupId;
+    // Right now the API doesn't support anything other than 'sv'
+    // const lang: string | undefined = req.query.lang;
+    const lang = "sv";
+    const dateStart: string | undefined = req.query.dateStart;
+    const dateEnd: string | undefined = req.query.dateEnd;
```

Comment:
Ideally, the router `Request` type would include the param types expected from the query using `Request<IParams>`.

Where `IParams` is something like
```typescript
interface IParams {
  userGroupId: number;
  lang: string;
  ...
}
```

Reply:
Turned out to be a bit of a hassle

Reply:
It did. We should probably keep it as is until Express' get better documentation for their typescript support.